src/the-2021-scheduler.md

```diff
@@ -60,22 +60,22 @@ following limits:
 
 Then:
 
-- If you click Grandchild3, you'll get up to 200 cards.
-- If you click Grandchild2, you'll get up to 5 cards.
-- If you click Grandchild1, you'll get up to 50 cards.
-- If you click Child, you'll get up to 30 cards from the Child deck and its
-  subdecks. No more than 5 cards will be taken from Grandchild2.
-- If you click Parent, you'll get up to 100 cards, with a maximum of 30
-  coming from Child and its subdecks.
+- If you click "Grandchild3", you'll get up to 200 cards.
+- If you click "Grandchild2", you'll get up to 5 cards.
+- If you click "Grandchild1", you'll get up to 50 cards.
+- If you click "Child", you'll get up to 30 cards from the "Child" deck and its
+  subdecks. No more than 5 cards will be taken from "Grandchild2".
+- If you click "Parent", you'll get up to 100 cards, with a maximum of 30
+  coming from "Child" and its subdecks.
```

Comment:
I think the quotation marks just clutter up this example, and make it hard to visually follow (especially up against all of those commas). If you don't like "click X" -- and I see you already changed it from "click on X" (which seems fine) -- try "select X" -- which uses the same language as the description above the example.